src/components/handbook-navigation/intra-page.js

```diff
@@ -3,7 +3,7 @@ import { Link } from "gatsby";
 import styled from "styled-components";
 
 const JoinCommunityWrapper = styled.div`
-  @media screen and (min-width: 751px) and (max-width: 1279px) {
+  @media screen and (min-width: 300px) and (max-width: 1279px) {
```

Comment:
is this by random you selected this resolution?

Reply:
I needed to make the mobile version responsive and most small mobiles are 300-350px in width